src/daemon/index.js

```diff
@@ -109,12 +109,8 @@ module.exports = async function (ctx) {
 
   ipcMain.on('ipfsConfigChanged', restartIpfs)
 
-  app.on('before-quit', async e => {
-    if (ipfsd) {
-      e.preventDefault()
-      await stopIpfs()
-      app.quit()
```

Comment:
not sure what was the purpose of this, but it seemed to interfere with `autoInstallOnAppQuit
` on Windows